db/migrate/20220524130709_remove_statuses_before_force_delete_from_domains.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveStatusesBeforeForceDeleteFromDomains < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :domains, :statuses_before_force_delete
```

Comment:
Please google one method. I don't remember the name, but I remember exactly what the method was. The idea is that this method checks whether the given column exists or not. If it doesn't exist, it skips the deletion and if it does, it deletes

Reply:
`column_exists?`

Reply:
yep